src/com/xilinx/rapidwright/util/ReplaceFrameData.java

```diff
@@ -653,15 +658,15 @@ public static void main(String[] args) {
 		// Run
 		if (op.equals("extract")) {
 
-			CodePerfTracker t = new CodePerfTracker("Elapsed time", false);
+			CodePerfTracker t = new CodePerfTracker("Extract frame data", false);

```

Comment:
The `if(op.equals("extract"))` and `else if (op.equals("replace"))` code bodies look very similar.  Can we reduce the duplication down to just the differences?

Reply:
It appears that way due to CodePerfTracker.  Because runtime is not important and it is within one second, I removed the use of CodePerfTracker.   In addition, I move reading bitstream and creating ReplaceFrameData objects out before the if (these are the true common code).